samples/openapi3/client/petstore/dart2/petstore_client_lib_fake/lib/model/inline_object2.dart

```diff
@@ -86,23 +86,22 @@ class InlineObject2EnumFormStringArrayEnum {
   const InlineObject2EnumFormStringArrayEnum._(this.value);
 
   /// The underlying value of this enum member.
-  final String value;
+  final List<String> value;
 
   @override
   bool operator ==(Object other) => identical(this, other) ||
-      other is InlineObject2EnumFormStringArrayEnum && other.value == value ||
-      other is String && other == value;
+      other is InlineObject2EnumFormStringArrayEnum && other.value == value;
```

Comment:
`other.value == value`

this is comparing references not contents of lists, so it's not what I would expect from `==` on enum. What are your expectations?

Reply:
Not a change for request, just asking.

Reply:
Yes this is a list but is actually completely wrong. This needs to be fixed in some other PR. An enum with a list as value is not valid. And the inline objects are gonna be removed soon.
I know what causes this but it will get too big.